### PR TITLE
apptest: improve routine that download tflite files @open sesame 03/27 17:58

### PR DESCRIPTION
**Pull request by leemgs**
Fixed issue https://github.com/nnsuite/nnstreamer/issues/1265.

The existing apptest module always download the same files such as
a network model and label file repeatedly. This commit is to improve
the existing operation to reduce a waiting time and save a network
cost of AWS instance.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

**Comment by leemgs** [comment]
Starting an aging test.

**Comment by jaeyun-jung** [comment]
[suggestion]
Currently app-test module uses only one model/label. 
I'm not sure that we may add another nn model for app test later. For this case, I suggest to make directory /tmp/nn_model/tflite_model and copy downloaded model.

**Comment by leemgs** [comment]
**Changelog**

    * Version 2:
    1. Added /tmp/nnsuite/ folder for more readability and maintenance
       - /tmp/nnsuite/tflite_model/ foldr in case of TF-lite model

    * Version 1:
    1. Added a routine that archive the network model files to save a network cost
       - AWS instance reboot at 5:00 AM everyday
       - apptest download the network files at once
       - Then apptest does not downoad the files repeatedly by using /tmp/nnsuite/* files.

**Comment by leemgs** [comment]
The aging test is successfully done. This PR is ready to be merged.

**Comment by leemgs** [comment]
> e.g., get_file_cached (URL, LINKPATH, FULLCACHEPATH)

Applied.

**Comment by leemgs** [comment]
**Changelog**
* V3
1. Added func_get_file_cached() to avoid duplicated statements.
2. Updated annotations

* V2
1. Added /tmp/nnsuite/ folder for more readability and maintenance
   - /tmp/nnsuite/tflite_model/ foldr in case of TF-lite model

* V1
1. Added a routine that archive the network model files to save a network cost
   - AWS instance reboot at 5:00 AM everyday
   - apptest download the network files at once
   - Then apptest does not downoad the files repeatedly by using /tmp/nnsuite/* files

**Comment by leemgs** [comment]
Starting an aging test.

**Comment by leemgs** [comment]
The aging test is successfully completed. The test is executed with https://github.com/nnsuite/nnstreamer/pull/1226.

**Comment by leemgs** [comment]
This PR is ready to be merged.

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 1]
```diff
@@ -143,19 +169,37 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
...
+
+
+    # Download a label file from the github.com
+    # For more details on /tmp folder, read /etc/default/rcS and /lib/systemd/system/systemd-tmpfiles-clean.service
     echo -e "" >> wget.log
     echo -e "[DEBUG] Downloading 'tflite label' with wget command ..." >> wget.log
-    wget -a wget.log https://github.com/nnsuite/testcases/raw/master/DeepLearningModels/tensorflow-lite/Mobilenet_v1_1.0_224_quant/labels.txt
-    result+=$?
-    cd ..
+    file_label="labels.txt"
```
Comment (jaeyun-jung) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: I think this line is unnecessary

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Nice catch. :)

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Updated.

**Pull request merged**